src/diffusers/configuration_utils.py

```diff
@@ -118,6 +118,35 @@ def register_to_config(self, **kwargs):
 
         self._internal_dict = FrozenDict(internal_dict)
 
+    def __getattr__(self, name):
+        """The only reason we overwrite `getattr` here is to gracefully deprecate accessing
+        config attributes directly. See https://github.com/huggingface/diffusers/pull/3129"""
+        try:
+            getattr(super(), name)
+        except AttributeError as e:
+            try:
+                is_in_config = self._safe_hasattr("_internal_dict") and hasattr(self._internal_dict, name)
+                is_attribute = self._safe_hasattr(name)
```

Comment:
Does this ever happen by the time `__getattr__` is called?

Reply:
Good point, think I might not need this second statement